cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -318,6 +320,7 @@ protected void configure() {
       bind(ProgramLifecycleService.class).in(Scopes.SINGLETON);
       bind(NamespaceAdmin.class).to(DefaultNamespaceAdmin.class).in(Scopes.SINGLETON);
       bind(NamespaceQueryAdmin.class).to(DefaultNamespaceQueryAdmin.class).in(Scopes.SINGLETON);
+      bind(SecureStoreService.class).to(DefaultSecureStoreService.class);
```

Comment:
ah! ok. But that means this is one more dependency that needs to be removed before moving secure store apis out of app-fabric service? Let's cross that bridge when we get there.